### PR TITLE
MM-8607 - document serverbusy endpoints

### PR DESCRIPTION
**Pull request by wiggin77**
This pull-request provides endpoint documentation for PR https://github.com/mattermost/mattermost-server/pull/13212

### CODE REVIEW

File: v4/source/system.yaml [hunk 1]
```diff
@@ -646,3 +646,90 @@
...
           $ref: '#/responses/Forbidden'
+
+  '/svrbusy':
+    post:
+      tags:
+        - system
+      summary: Set the server busy (high load) flag
+      description: |
+        Marks the server as currently having high load which disables non-critical services such as search, statuses and typing notifications.
+        ##### Permissions
```
Comment (jespino) [inline v4/source/system.yaml]: I would add here, and in the other endpoints that the minimum server version is going to be 5.20

File: v4/source/system.yaml [hunk 2]
```diff
@@ -646,3 +646,90 @@
...
+        '200':
+          description: Server busy flag set successfully
+          schema:
+            $ref: "#/definitions/StatusOK"
+        '400':
+          $ref: '#/responses/BadRequest'
+        '403':
+          $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
```
Comment (jespino) [inline v4/source/system.yaml]: Can you add the `lang: 'curl'` option here?

File: v4/source/system.yaml (hunk 1 above)
Comment (wiggin77) [reply v4/source/system.yaml]: Minimum server version 5.20 added to all three endpoints via https://github.com/mattermost/mattermost-api-reference/commit/e26aa85745a0df9b21b3b642d3f7efeb1b05de95

File: v4/source/system.yaml (hunk 2 above)
Comment (wiggin77) [reply v4/source/system.yaml]: curl sample added to all three endpoints via https://github.com/mattermost/mattermost-api-reference/commit/e26aa85745a0df9b21b3b642d3f7efeb1b05de95 

curl samples tested as well.

**Comment by hanzei** [comment]
I'm inclined to merge this PR now given that it's a new endpoint and its' clearly documented that this is for 5.20. Are you fine with this @reflog @jespino?

**Comment by reflog** [comment]
Cool by me, maybe ping @amyblais to confirm?

**Comment by amyblais** [comment]
@hanzei Is there a reason we want to merge this now instead of wait? It should be fine as long as there is no risk of confusion.

**Comment by hanzei** [comment]
The reasons I see are:
1. We tent do forget merging these PRs.
2. The API endpoint is in the code now and devs might want to use it right now or know about it
3. To avoid merge conflicts.

**Pull request merged**